infra/dataindex_and_jobservice/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+resources:
+- ../dataindex
+- 03-jobservice.yml
+
+configMapGenerator:
+  - name: jobs-service-properties
```

Comment:
I think we can have a true ConfigMap with an `application.properties` file and mount it at `deployment/config`, wdyt? So we won't rely on EnvVars.

Reply:
added